tests/test_client.py

```diff
@@ -0,0 +1,120 @@
+import re
+import json
+from six.moves.urllib.parse import parse_qs
+import pytest
+from urlobject import URLObject
+import responses
+from nylas.client import APIClient
+
+
+def urls_equal(url1, url2):
+    """
+    Compare two URLObjects, without regard to the order of their query strings.
+    """
+    return (
+        url1.without_query() == url2.without_query() and
+        url1.query_dict == url2.query_dict
+    )
+
+
+def test_custom_client():
+    # Can specify API server
+    custom = APIClient(api_server="http://example.com")
+    assert custom.api_server == "http://example.com"
```

Comment:
The exception message says that you must use `https://`, but doesn't actually require it...